extensions/micrometer/deployment/src/test/java/io/quarkus/micrometer/deployment/binder/StorkMetricsServiceDiscoveryFailTest.java

```diff
@@ -0,0 +1,106 @@
+
+package io.quarkus.micrometer.deployment.binder;
```

Comment:
It seems that this is failing on windows. Because it uses Redis container (and windows does not have containers), we should disable it on windows.